src/main/java/seedu/address/logic/commands/BorrowCommand.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.logic.commands;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.Debt;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+//@@author jelneo
+/**
+ * Updates debt field by amount that has been repaid by person
+ */
+public class BorrowCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "borrow";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": increase the debt of a person by "
+            + "the amount of money entered.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "AMOUNT (must have at least 1 digit and either a positive integer or a positive number with "
+            + "two decimal places)\n"
+            + "Example: " + COMMAND_WORD + " 1 120.50";
```

Comment:
You wanted a comma is it jel? Like "1,120.50" ?

Reply:
1 is the index of the person who borrowed extra cash